src/factory.ts

```diff
@@ -111,7 +118,15 @@ export async function buildStrategy(
     bumpMinorPreMajor: options.bumpMinorPreMajor,
     bumpPatchForMinorPreMajor: options.bumpPatchForMinorPreMajor,
   });
-  const strategyOptions: StrategyOptions = {
+  const changelogNotes = buildChangelogNotes({
+    type: options.changelogType || 'default',
+    github: options.github,
+    changelogSections: options.changelogSections,
+    // commitPartial: options.commitPartial,
```

Comment:
Do we need these commented out, can they just be deleted?

Reply:
We can probably delete. Those are options used by ruby to customize the changelog templates and we considering exposing them to the configuration options but did not.